cleanlab/datalab/internal/issue_manager_factory.py

```diff
@@ -79,23 +88,40 @@ class _IssueManagerFactory:
     """Factory class for constructing concrete issue managers."""
 
     @classmethod
-    def from_str(cls, issue_type: str) -> Type[IssueManager]:
+    def from_str(cls, issue_type: str, task: str = "classification") -> Type[IssueManager]:
```

Comment:
The default `task = "classification"` should be set at one place in the code base, in the constructor of Datalab:
https://github.com/cleanlab/cleanlab/blob/c5af24340d6cdccc5caf1d76b11219016482d900/cleanlab/datalab/datalab.py#L103


```suggestion
    def from_str(cls, issue_type: str, task: str) -> Type[IssueManager]:
```